component/src/main/java/me/androidbox/component/agenda/AgendaHeader.kt

```diff
@@ -0,0 +1,127 @@
+package me.androidbox.component.agenda
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.ui.theme.*
+
+@Composable
+fun AgendaHeader(
+    modifier: Modifier = Modifier,
+    agendaHeaderItem: AgendaHeaderItem,
+    subTitle: String
```

Comment:
modifier should be the first default parameter

If not, this makes it so to set the agendaHeaderItem and subTitle you'd need to use named parameters